pkg/agent/flowexporter/connections/baseconnections.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/proxy"
+)
+
+type BaseConnectionStore interface {
+	FillPodInfo(conn *flowexporter.BaseConnection)
+	FillServiceInfo(conn *flowexporter.BaseConnection, serviceStr string)
+}
+
+type baseConnectionStore struct {
+	ifaceStore    interfacestore.InterfaceStore
+	antreaProxier proxy.Proxier
+}
```

Comment:
this is not storing any connections, why is it a "connection store"?

when I suggested re-using code, I meant between what is currently `connectionStore` and what is currently `denyConnectionStore`, in other words the types which are currently used to store connections. Notice how they have methods to get a connection, add a connection, iterate over all connections and invoke a callback, etc. Also how they both use `flowexporter.ConnectionKey` to index connections. The current `BaseConnectionStore` is really the tip of the iceberg and doesn't actually manipulate connections.